.gitignore

```diff
@@ -1,4 +1,7 @@
 _site
 CNAME
 .sass-cache
-.jekyll-metadata
\ No newline at end of file
+.jekyll-metadata
+_posts/2016-10-07-melody-lam copy.md
+
+_posts/2016-11-02-gabby-Ibanez-Dacruz.md
```

Comment:
This blog post is missing from this update because of this line. Please remove and recommit.